storybook/src/components/Spotlight/Spotlight.stories.tsx

```diff
@@ -29,6 +36,38 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
+export const StickOut: Story = {
+  render: ({ as, color }) => (
+    <Grid>
+      <Grid.Cell
+        span="all"
+        rowSpan={{ narrow: 2, medium: 2, wide: 1 }}
+        rowStart={2}
+        style={{
+          marginBlock: 'calc(var(--ams-space-grid-md) * -1)',
+          marginInline: 'calc(var(--ams-space-grid-lg) * -1)',
+        }}
+      >
+        <Spotlight as={as} color={color} style={{ height: '100%' }} />
+      </Grid.Cell>
+      <Grid.Cell rowStart={{ narrow: 3, medium: 3, wide: 2 }} span={{ narrow: 4, medium: 8, wide: 6 }} start={1}>
```

Comment:
This `start={1}` is often optional, but required here to place the 3 items correctly.